seahub/thumbnail/utils.py

```diff
@@ -6,6 +6,8 @@
 import urllib2
 import logging
 from StringIO import StringIO
+from PIL import Image, ImageCms
+from psd_tools import PSDImage
```

Comment:
It is about psd_tools. Is it bundled or installed by default? Otherwise I think this needs documentation how to set it up.